modules/nw-cno-view.adoc

```diff
@@ -33,15 +33,15 @@ Spec: <1>
   Cluster Network:
     Cidr:         10.128.0.0/14
     Host Prefix:  23
-  Network Type:   OpenShiftSDN
+  Network Type:   OVN-Kubernetes
   Service Network:
     172.30.0.0/16
 Status: <2>
   Cluster Network:
     Cidr:               10.128.0.0/14
     Host Prefix:        23
   Cluster Network MTU:  8951
```

Comment:
I don't know if this is the same on OVN-Kubernetes; might need to re-run this for output.

Reply:
yeah, it would be 8901

Reply:
I pulled this from a 4.17 gcp cluster. I got 1360 for MTU 🤷🏻‍♂️ .  Maybe I did something wrong? @danwinship

Reply:
It varies depending on the platform / NICs. The original values were probably from AWS. Anyway, ovn-k MTU is always 50 less than openshift-sdn MTU[*], which I why I said it should change from 8951 to 8901. "8901" is an MTU value you would see in _some_ ovn-k-based clusters, but "8951" is not.

[*] unless you enable IPSEC in which case it will be even less

Reply:
Didn't realize the ovn-k -50 rule. Is this ok the way I have it or should I change to 8901?